src/go/unit/nxt_cgo_lib.c

```diff
@@ -17,9 +17,9 @@ static nxt_cgo_str_t *nxt_cgo_str_init(nxt_cgo_str_t *dst,
 static int nxt_cgo_add_port(nxt_unit_ctx_t *, nxt_unit_port_t *port);
 static void nxt_cgo_remove_port(nxt_unit_ctx_t *, nxt_unit_port_id_t *port_id);
 static ssize_t nxt_cgo_port_send(nxt_unit_ctx_t *, nxt_unit_port_id_t *port_id,
-    const void *buf, size_t buf_size, const void *oob, size_t oob_size);
```

Comment:
I see no reason to rename formal parameters. It looks consistent enough: buf - buf_size, oob - oob_size.

Reply:
Ok, I changed because the name is `oobn` in other places.

Reply:
Well, I'm not insist to use `oob_size` in new code, but here in prototype, `oobn` looks not consistent and such changes increase patch size without adding any benefit.

Reply:
got it